docs/api/settings.md

```diff
@@ -0,0 +1,283 @@
+REST API: Settings
+==================
+
+The settings REST API allows you to view and update WooCommerce Stripe settinngs.
```

Comment:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/314/commits/7abd2c6e22a15feadfdfed0282db71601a872bd4.